src/main/java/liquibase/ext/neo4j/change/InsertNodeChange.java

```diff
@@ -0,0 +1,72 @@
+package liquibase.ext.neo4j.change;
+
+import liquibase.change.ChangeMetaData;
+import liquibase.change.ColumnConfig;
+import liquibase.change.DatabaseChange;
+import liquibase.change.core.InsertDataChange;
+import liquibase.database.Database;
+import liquibase.exception.ValidationErrors;
+import liquibase.ext.neo4j.database.Neo4jDatabase;
+import liquibase.statement.SqlStatement;
+import liquibase.statement.core.RawParameterizedSqlStatement;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static liquibase.ext.neo4j.change.ColumnMapper.mapValue;
+
+@DatabaseChange(name = "insert", priority = ChangeMetaData.PRIORITY_DATABASE,
+        description = "Inserts a node to the graph")
+public class InsertNodeChange extends InsertDataChange {
+
+    private String labelName = "";
+
+    @Override
+    public ValidationErrors validate(Database database) {
+        ValidationErrors errors = new ValidationErrors(this);
+        if (!(database instanceof Neo4jDatabase)) {
+            errors.addError(String.format(
+                    "unsupported database %s, expected Neo4j",
+                    database == null ? "null" : database.getShortName()
+            ));
+        }
+        if (labelName.isEmpty()) {
+            errors.addError("missing label name for insert");
+        }
+        return errors;
+    }
+
+    public void setLabelName(String labelName) {
+        this.setTableName(labelName);
```

Comment:
Good use of overriding the tableName as labelName. Then users who use the regular `<insert tableName="x">` version will still work.

In theory it would be nice to be able to have `<insert neo4j:labelName="x">` but that requires a liquibase-core XSD changes and would remove the "required" intellisense for everyone else, so I'm good with having it as a completely different element definition.

Reply:
Fixed in 99cafd81900bb80d2c33ed9a94cb237b0b7bcddd